scripts/Octave/SyncChirpProcessor_CoeffcientsMatrixTest/powersin_ampl.m

```diff
@@ -1,5 +1,5 @@
 function C = powersin_ampl(N, A)
-    % Code from http://ant-novak.com/Nonlinear_System_Identification_Waveguide.php
+    % Code from https://ant-novak.com/Nonlinear_System_Identification_Waveguide.php
```

Comment:
HTTPS doesn't work here.

Reply:
I guess he modified his site. That should be  https://ant-novak.com/pages/sss/  ???

Reply:
Don't know. That's copy-paste from feedback of user.

Reply:
It looks really really close, so I fixed the link. At least it is not broken